lua/roslyn/init.lua

```diff
@@ -209,16 +172,14 @@ function M.setup(config)
     local default_config = {
         filewatching = true,
         exe = nil,
-        -- I don't know how to make this a partial type.
         ---@diagnostic disable-next-line: missing-fields
         config = {
+            -- setting default capabilities early allows users to provide their own
             capabilities = get_default_capabilities(config),
         },
     }
 
-    ---@type InternalRoslynNvimConfig
```

Comment:
Maybe add this type back? Unless the language server is infering this. On mobile now so don't have time to check now

Reply:
yeah, removed because lua ls is inferring properly. Can add though if you really want ;)

Reply:
No it's okay then😄